modules/kubernetes-addons/argo-rollouts/locals.tf

```diff
@@ -0,0 +1,23 @@
+locals {
+
+  name                 = "argo-rollouts"
+  service_account_name = "${local.name}-sa"
+  default_helm_config = {
+    name        = local.name
+    chart       = local.name
+    repository  = "https://argoproj.github.io/argo-helm"
+    version     = "2.9.1"
+    namespace   = local.name
+    description = "Argo Rollouts AddOn Helm Chart"
+  }
+
+  helm_config = merge(
+    local.default_helm_config,
+    var.helm_config
+  )
```

Comment:
We may need to add forced `set_values` and `set_sensitive_values` to this merge command. This will be the last variable in the order of the merge.

Reply:
addressed